README.md

```diff
@@ -30,9 +31,9 @@ You should be able to navigate to http://localhost/_pages/sample-article.html to
 
 ### Accessing The Edit UI
 
-The edit interface of Clay is a component itself called [`Kiln`](https://github.com/clay/clay-kiln)! To begin editing with the UI you'll need to make sure you've run `make boostrap-user` after replacing the sample user with your own Gmail address.
+The edit interface of Clay is a component itself called [`Kiln`](https://github.com/clay/clay-kiln)! To begin editing with the UI you'll need to make sure you've run `make boostrap-user` after replacing the sample user with your own credentials or using the default one.
```

Comment:
It might be helpful to have an example of a username/provider, just so folks know what sort of thing they'll have to include.